app/config.py

```
@@ -0,0 +1,88 @@
+    api_port: int = 8000
+    
+    model_config = SettingsConfigDict(
+        env_file=".env",
+        case_sensitive=False,
+        env_file_encoding="utf-8",
+    )
+    
+    def get_allowed_mailfrom_list(self) -> list[str]:
+        """Parse comma-separated allowed MailFrom addresses"""
+        if not self.allowed_mailfrom or not self.allowed_mailfrom.strip():
+            raise ValueError("ALLOWED_MAILFROM configuration is required and cannot be empty")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)